README.md

```diff
@@ -1,54 +1,57 @@
-# Cleaner
+# Outlier Detector
 
-|                |                                       |
-| -------------- | ------------------------------------- |
-| Name           | Cleaner                               |
-| Version        | v1.0.0                                |
+|                |                                                                       |
+| -------------- | --------------------------------------------------------------------- |
+| Name           | Outlier Detector                                                      |
+| Version        | v2.0.0                                                                |
 | Dockerhub Link | [weevenetwork/cleaner](https://hub.docker.com/r/weevenetwork/cleaner) |
```

Comment:
```suggestion
| Dockerhub Link | [weevenetwork/outlier-detector](https://hub.docker.com/r/weevenetwork/outlier-detector) |
```

Reply:
We should have a new DockerHub repo for that. Probably I will delete the cleaner repo later.